feature/security/gnsi/credentialz/tests/hiba_authentication/hiba_authentication_test.go

```diff
@@ -0,0 +1,581 @@
+package hiba_authentication
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/gnsi/credentialz"
+	tpb "github.com/openconfig/kne/proto/topo"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/binding"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	username                 = "testuser"
+	userPrivateKeyFilename   = "users/testuser"
+	userCertFilename         = "users/testuser-cert.pub"
+	dutPrivateKeyFilename    = "hosts/dut"
+	dutCertFilename          = "hosts/dut-cert.pub"
+	caPublicKeyFilename      = "ca.pub"
+	hostCertificateVersion   = "v1.0"
+	hostCertificateCreatedOn = 1705962293
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+func setupUser(t *testing.T, dut *ondatra.DUTDevice) {
```

Comment:
as in the other credentialz tests prs, please move this all to a utility library. (all of this == 'except the actual Test.... functions)

Reply:
Added common library in `internal/security/credz/credz.go`